src/python/pants/backend/python/dependency_inference/import_parser.py

```diff
@@ -99,20 +119,49 @@ def visit_ImportFrom(self, node) -> None:
         for alias in node.names:
             self.explicit_imports.add(f"{abs_module}.{alias.name}")
 
-
-class _Py27AstVisitor(ast27.NodeVisitor, _BaseAstVisitor):
     def visit_Str(self, node) -> None:
         val = ensure_text(node.s)
-        self.maybe_add_inferred_import(val)
+        self._maybe_add_inferred_import(val)
 
 
-class _Py3AstVisitor(ast3.NodeVisitor, _BaseAstVisitor):
-    def visit_Str(self, node) -> None:
-        self.maybe_add_inferred_import(node.s)
+class _CSTVisitor(cst.CSTVisitor, VisitorInterface):
+    def __init__(self, module_name: str) -> None:
+        self._module_parts = module_name.split(".")
+        self.explicit_imports: Set[str] = set()
+        self.inferred_imports: Set[str] = set()
+
+    @classmethod
+    def visit_tree(cls: Type[_Visitor], tree: Any, module_name: str) -> _Visitor:
+        visitor = cls(module_name)
+        tree.visit(visitor)
+        return visitor
 
+    def _maybe_add_inferred_import(self, s: Union[str, bytes]) -> None:
+        if isinstance(s, bytes):
+            return
```

Comment:
This is because `SimpleString` includes bytes, I presume?

Reply:
Yes, although it's not explicitly mentioned in the documentation: https://libcst.readthedocs.io/en/latest/nodes.html?highlight=import#libcst.SimpleString.
 
With pex:
```python
>>> cst.parse_expression('b"asdf"').evaluated_value
b'asdf'
>>> cst.parse_expression('"asdf"').evaluated_value
'asdf'
```